indico/modules/rb_new/client/js/common/timeline/MonthlyTimelineContent.jsx

```diff
@@ -73,38 +81,49 @@ export default class MonthlyTimelineContent extends WeeklyTimelineContent {
     renderDividers() {
         const nDays = this.dates.length;
         const daySize = (100 / nDays);
-
         const {dateRange} = this.props;
+        const weekendDays = this.getWeekendDays();
```

Comment:
This causes an error. IT should be `this.weekendDays`.

Reply:
Indeed [(getter docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get)

Reply:
true, just forgot to change it here.